src/main/cpp/vision/DragonVision.cpp

```diff
@@ -657,15 +647,14 @@ std::optional<frc::Pose3d> DragonVision::GetAprilTagPose(FieldConstants::AprilTa
 	return std::nullopt;
 }
 
-void DragonVision::SetPipeline(DRAGON_LIMELIGHT_CAMERA_USAGE position, DRAGON_LIMELIGHT_PIPELINE pipeline)
+void DragonVision::SetPipeline(DRAGON_LIMELIGHT_CAMERA_USAGE position, DRAGON_LIMELIGHT_PIPELINE pipeline) // TODO: When we rework vision, we should make this function a result of Camera names not usage, this way we can specify the camera we want. Not all cameras of a certain change to the same pipeline
```

Comment:
Disagree with this comment; outside world shouldn't know camera names.  Extra usages should have been added to account for the combinations which apparently never happened when non-apriltag usages such as algae were added.